h2o-py/h2o/utils/compatibility.py

```diff
@@ -5,7 +5,6 @@
 This module gathers common declarations needed to ensure Python 2 / Python 3 compatibility.
```

Comment:
Is this file can be removed totally in the future?

Reply:
yes, thinks so, but it will require some additional refactoring. (I've tried to be minimalistic with this change as possible :-D)

Reply:
Sure :)